eslint-plugin-skyscanner-no-axios/src/rules/no-axios.test.js

```diff
@@ -0,0 +1,55 @@
+const { RuleTester } = require('eslint');
+
+const noAxios = require('./no-axios');
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-axios', noAxios, {
```

Comment:
I would also add a test for named imports and for requires with property access
`require('axios').default`